builds/android/gradle.properties

```diff
@@ -1,3 +1,8 @@
 android.enableJetifier=true
 android.useAndroidX=true
-android.useDeprecatedNdk=true
+
+VERSION_NAME=1.0
```

Comment:
We should start to set these with each release from now on.
This way f-droid will have it easier at least.